events/jackson2018.md

```diff
@@ -0,0 +1,40 @@
+| 10:40 | Break                                        |             |  0:20  |
+| 11:00 | [Introduction to NumPy](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/NumPy/Numpy%20Basics.ipynb)                        | Ryan        |  1:00  |
+| 12:00 | [Introduction to MetPy](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Metpy_Introduction/Introduction%20to%20MetPy.ipynb)                        | John        |  0:30  |
+| 12:30 | Lunch                                        |             |  1:15  |
+| 13:45 | [Introduction Siphon](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Siphon/Siphon%20Overview.ipynb)                          | Ryan        |  1:00  |
+| 14:45 | Break                                        |             |  0:15  |
+| 15:00 | [Upper Air Data Analysis and the Skew-T](https://github.com/Unidata/unidata-python-workshop/tree/master/notebooks/Skew_T)       | Jon         |  1:00  |
+| 16:00 | Break                                        |             |  0:15  |
+| 16:15 | [Making Maps with Cartopy](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/CartoPy/CartoPy.ipynb)                     | Ryan        |  1:00  |
+| 17:15 | End of day 1                                 |             |        |
+
+## Friday, August 31st
+
+|  Time | Section                                      | Presenter   | Length |
+|-------|----------------------------------------------|-------------|--------|
+|  9:00 | [Advanced MetPy](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/MetPy_Advanced/Isentropic%20Analysis.ipynb)                               | Ryan        |  1:00  |
+| 10:00 | Break                                        |             |  0:15  |
+| 10:15 | [Working with Satellite Data](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Satellite_Data/Working%20with%20Satellite%20Data.ipynb)                  | John        |  1:00  |
+| 11:15 | Break                                        |             |  0:15  |
+| 11:30 | [Surface Data Analysis](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Surface_Data/Surface%20Data%20with%20Siphon%20and%20MetPy.ipynb)                        | Ryan        |  1:00  |
+| 12:30 | Lunch                                        |             |  1:15  |  
+| 13:45 | [Model Data](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Model_Output/Downloading%20model%20fields%20with%20NCSS.ipynb)                                   | Ryan        |  1:00  |
+| 14:45 | Break                                        |             |  0:15  |
+| 15:00 | [Time Series Analysis of Buoy Data](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Time_Series/Basic%20Time%20Series%20Plotting.ipynb)            | Hailey      |  1:00  |
+| 16:00 | End of day 2                                 |             |        |
```

Comment:
Our flight is at 6:17 - should we plan on ending early here?

Reply:
Rough sketch, we can adjust later...eh, I'll bump to 3:45.